src/main/java/org/broadinstitute/hellbender/tools/copynumber/coverage/caller/CalledCopyRatioSegment.java

```diff
@@ -0,0 +1,64 @@
+package org.broadinstitute.hellbender.tools.copynumber.coverage.caller;
+
+import org.broadinstitute.hellbender.tools.copynumber.coverage.segmentation.CopyRatioSegment;
+import org.broadinstitute.hellbender.utils.Utils;
+
+public class CalledCopyRatioSegment extends CopyRatioSegment {
+    public enum Call {
+        AMPLIFICATION_CALL("+"),
+        DELETION_CALL("-"),
+        NEUTRAL_CALL("0");
+
+        private final String outputString;
+
+        Call(final String outputString) {  this.outputString = outputString; }
+
+        public String getOutputString() {
+            return outputString;
+        }
+    }
+
+    private final Call call;
+
+    public CalledCopyRatioSegment(final CopyRatioSegment segment,
+                                  final Call call) {
+        super(segment.getInterval(), segment.getNumPoints(), segment.getMeanLog2CopyRatio());
+        this.call = Utils.nonNull(call);
+    }
+
+    public Call getCall() {
+        return call;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
```

Comment:
This would look better as an `else if` on the previous line.  Ditto for the next one.

Reply:
This is (slightly cleaned up) IntelliJ auto-generated code, so in the interest of consistency I'll leave it.